### ISSUE TITLE
NirLauncher relying on Chocolatey shims causes some weird issues

### ISSUE THREAD

**Issue by God-damnit-all** [majkinetor/au-packages#235]
I've noticed things that shouldn't be launching with a command line keeping a command line window open, and this also makes NirLauncher's Explorer Copy feature worthless since it will just copy the shim instead.

I think it would be better to have the sysinternals nlp in the Launcher directory and insert the direct path to the tools into the nlp file. This can be programatically determined by running one of the Sysinternals shims with `--shimgen-noop ` and using the second line, the one that says `path to executable:`. Running on each and every shim shouldn't be necessary, just taking the path from one and using it for the rest would suffice.

**Comment by majkinetor** [comment]
Yeah, I can do that when the package gets allowed to proceed. I wouldn't count on that though, but there is nothing I can do.
